sdk/mobile-center-distribute/src/main/java/com/microsoft/azure/mobile/distribute/DeepLinkActivity.java

```diff
@@ -25,14 +26,19 @@ public void onCreate(Bundle savedInstanceState) {
         String requestId = intent.getStringExtra(EXTRA_REQUEST_ID);
         String distributionGroupId = intent.getStringExtra(EXTRA_DISTRIBUTION_GROUP_ID);
         String updateToken = intent.getStringExtra(EXTRA_UPDATE_TOKEN);
+        String updateSetupFailed = intent.getStringExtra(EXTRA_UPDATE_SETUP_FAILED);
         MobileCenterLog.debug(LOG_TAG, getLocalClassName() + ".getIntent()=" + intent);
         MobileCenterLog.debug(LOG_TAG, "Intent requestId=" + requestId);
         MobileCenterLog.debug(LOG_TAG, "Intent distributionGroupId=" + distributionGroupId);
         MobileCenterLog.debug(LOG_TAG, "Intent updateToken passed=" + (updateToken != null));
+        MobileCenterLog.debug(LOG_TAG, "Intent updateSetupFailed passed=" + (updateSetupFailed != null));
 
         /* Store redirection parameters if both required values were passed. */
         if (requestId != null && distributionGroupId != null) {
             Distribute.getInstance().storeRedirectionParameters(requestId, distributionGroupId, updateToken);
+        } else if (updateSetupFailed != null){
```

Comment:
Not sure if this is the only true condition for `updateSetupFailed` case. Is it possible that `distributionGroupId ` could be also `true` in this case? Please take a look at this.

Reply:
this should be fine, if the in-app updates setup fails, there won't be any query parameters except for updateSetupFailed and requestId